igor-web/src/main/groovy/com/netflix/spinnaker/igor/travis/service/TravisResultConverter.groovy

```diff
@@ -48,6 +48,19 @@ class TravisResultConverter {
                 throw new IllegalArgumentException("state: ${state} is not known to TravisResultConverter.")
                 break
         }
+    }
 
+    static Boolean running(String state) {
+        switch (state) {
+            case "created":
```

Comment:
yes, as far as I know. I have never seen [this] (https://github.com/spinnaker/igor/blob/master/igor-web/src/main/groovy/com/netflix/spinnaker/igor/travis/service/TravisResultConverter.groovy#L26) code fail.